pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools>=43.0.0", "wheel"]
```

Comment:
Why would we remove the pin on `setuptools` in `test_requirements.txt` but add it here? Do we want to pin this and only pin it in one place to avoid conflicts or something else? And should we really pin `setuptools` at all?